bcmath/bcmath.php

```diff
@@ -150,13 +151,22 @@ function bcpow (string $num, string $exponent, ?int $scale = 0): string
 function bcsqrt (string $num, ?int $scale)
 {}
 
+#[PhpStormStubsElementAvailable(to: '7.2')]
 /**
  * Set default scale parameter for all bc math functions
  * @link https://php.net/manual/en/function.bcscale.php
- * @param int|null $scale <p>
- * The scale factor. Since 7.3.0 can be omitted.
- * </p>
- * @return int|true <b>INT</b> since 7.3.0 and <b>TRUE</b> before.
+ * @param int $scale
+ * @return true
+ */
+function bcscale (int $scale): bool
+{}
+
+#[PhpStormStubsElementAvailable('7.3')]
```

Comment:
This should be `8.0`.

Reply:
According to docs - yes, but according to https://3v4l.org/Ptm7Q seems that null can be passed successfully even with versions < 8 so I think it makes sense just to make type nullable and apply LanguageLevelTypeAware for return type only according to https://3v4l.org/o0UXs

Reply:
@voku @MaXal what do you think about such solution? Leave just one function with nullable parameter and return type that will have appropriate LanguageLevelTypeAware attribute.

Reply:
@isfedorov https://3v4l.org/HP28H I think PHP 8.0 will be correct here

Reply:
I added three different versions. I hope "PhpStormStubsElementAvailable" is used correctly  now.